doc/reference/widgets/ChatBox.md

```diff
@@ -0,0 +1,191 @@
+# ChatBox
+
```

Comment:
There is an issue with the button height and position/ alignment in the `MaterialTemplate`. Its the same with the `FastListTemplate`. There might be issues with other templates, but have not tested.

![image](https://github.com/holoviz/panel/assets/42288570/02bebeca-1d31-4f12-a73a-bff73b70619d)


```python
import panel as pn
from panel.widgets import ChatInterface, ChatEntry

pn.extension()


def even_or_odd(contents, user, instance):
    if len(contents) % 2 == 0:
        return "Even number of characters."
    return "Odd number of characters."


chat = ChatInterface(
    callback=even_or_odd,
    value=[
        ChatEntry(value="Hi", user="Assistant", avatar="🤖"),
        ChatEntry(value="I have so many questions to ask!", user="User", avatar="😊"),
    ],
)

pn.template.MaterialTemplate(main=[chat], theme="dark").servable()
```

Reply:
Thanks for spotting; it's fixed now by using panel-surface-color
<img width="1119" alt="image" src="https://github.com/holoviz/panel/assets/15331990/8d2f7e2c-b261-4bb3-b24f-1a8e8c598bb8">